app/assets/javascripts/controllers/container_deployment/dialogs.module.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @name  MiQ Dialogs
+ *
+ * @description
+ *   Dialogs module for MiQ.
+ *
+ */
+angular.module('miq.dialogs', []);
```

Comment:
Definitely agreed on consistency, even though it may be hard to find any now :). I'll leave this up to you, if you can figure out a more fitting name, feel free, if not, introducing some consistency with modules is on my list anyway :)